web_external/User/invitationConfirmationView.js

```diff
@@ -0,0 +1,27 @@
+isic.views.InvitationConfirmationView = isic.View.extend({
```

Comment:
Oops, it looks like there is a `.styl` file for this.

Just the filename should be updated.